src/main/java/io/jenkins/plugins/synopsys/security/scan/service/scan/coverity/CoverityParametersService.java

```diff
@@ -212,6 +231,42 @@ private void setCoverityPolicyView(Map<String, Object> coverityParameters, Cover
         }
     }
 
+    private void setBuildCommand(Map<String, Object> coverityParameters, Coverity coverity) {
+        String value = coverityParameters
+                .get(ApplicationConstants.COVERITY_BUILD_COMMAND_KEY)
+                .toString()
+                .trim();
+        if (!value.isBlank()) {
+            Build build = new Build();
+            build.setCommand(value);
+            coverity.setBuild(build);
+        }
+    }
+
+    private void setCleanCommand(Map<String, Object> coverityParameters, Coverity coverity) {
+        String value = coverityParameters
+                .get(ApplicationConstants.COVERITY_CLEAN_COMMAND_KEY)
+                .toString()
+                .trim();
+        if (!value.isBlank()) {
+            Clean clean = new Clean();
+            clean.setCommand(value);
+            coverity.setClean(clean);
+        }
+    }
+
+    private void setConfigCommand(Map<String, Object> coverityParameters, Coverity coverity) {
+        String value = coverityParameters
+                .get(ApplicationConstants.COVERITY_CONFIG_PATH_KEY)
+                .toString()
+                .trim();
+        if (!value.isBlank()) {
+            Config config = new Config();
+            config.setPath(value);
+            coverity.setConfig(config);
+        }
+    }
+
```

Comment:
Add some unit tests to cover **Lines 174-268**

Reply:
Done